foundation/auth/src/idtoken.rs

```diff
@@ -68,7 +68,7 @@ pub async fn create_id_token_source(
     }
 }
 
-async fn id_token_source_from_credentials(
+pub(crate) async fn id_token_source_from_credentials(
```

Comment:
@yoshidan Not sure about your preference in term of visibility ?

Reply:
The scope you specified is fine.